### ISSUE TITLE
Missing point data written as zeros in python

### ISSUE THREAD

**Issue by aclouthier** [pyomeca/ezc3d#128]
The point data I am trying to write contains some nans to represent missing markers. These get set to 0 when I write the c3d file in Python. Is there a way to have these be recognized as not visible in the c3d file rather than being set to 0?

**Comment by pariterre** [comment]
Hello @aclouthier !

When you say that they are set to 0, do you mean if you read them in another software or if you read them back using ezc3d? I've had a quick try with the "Vicon.c3d" test file which contains nan:
```python
import ezc3d

c3d = ezc3d.c3d("Vicon.c3d")
array = c3d["data"]["points"]
print(array[:, :, 120])  # This shows nan at third to last position
c3d.write("Coucou.c3d")  # Write the file

c3d_2 = ezc3d.c3d("Coucou.c3d")  # Read that back
array_2 = c3d_2["data"]["points"]
print(array_2[:, :, 120])  # This still shows nan at third to last position
```

But I see where they are set to 0 in Python, so maybe if you read them in an external software, it puts them to zero. In that case, I can make a quick fix for that!

**Comment by aclouthier** [comment]
I just checked, and if I read it back in using ezc3d, they are nan. But if I open it in another software they are zero. I tried Visual3D and Mokka and it is 0 in both. 

Thanks for your help!

**Comment by pariterre** [comment]
The PR #129 should fix the problem!
If you are compiling ezc3d yourself, it will be available in minutes. If you have installed using conda, please tell me, I will then trigger a new release so you can install the last version easily :)

**Comment by aclouthier** [comment]
I installed using conda. Thanks again!

**Comment by pariterre** [comment]
Ok, I'll make a Release, this takes some time for all the things to update, so count 1 to 2 hours :)

Pleased to have helped!

**Comment by aclouthier** [comment]
Hi again! I noticed it was still writing the missing markers as 0s, but if I remove the if statement that is supposed to set the nans it seems to work and the nan markers are missing when I open the c3d in Mokka. 

            for i in range(nb_points):
               # if np.isnan(data_points[:, i, f]).any():
               #     pt.set(np.nan, np.nan, np.nan,-1)
               # else:
                pt.set(data_points[0, i, f], data_points[1, i, f], data_points[2, i, f])
                pt.residual(data_meta_points["residuals"][0, i, f])
                pt.cameraMask(data_meta_points["camera_masks"][:, i, f].tolist())
                pts.point(pt, i)

**Comment by pariterre** [comment]
I was maybe overthinking it!
Can you Pull Request this modification? So you get the credit for the fix :)

**Comment by aclouthier** [comment]
Ok, I made a pull request. Let me know if I've done that wrong though, I've never done a pull request before to be honest!

**Comment by pariterre** [comment]
Fixed by @aclouthier  in #134 
GG
